### ISSUE TITLE
Not all folders configurable from e107_config.php by default

### ISSUE THREAD

**Issue by nlstart** [e107inc/e107#3774]
In install.php as from line 1368 these settings are being written to the e107_config.php file:
```
\$ADMIN_DIRECTORY     = '{$this->e107->e107_dirs['ADMIN_DIRECTORY']}';
\$FILES_DIRECTORY     = '{$this->e107->e107_dirs['FILES_DIRECTORY']}';
\$IMAGES_DIRECTORY    = '{$this->e107->e107_dirs['IMAGES_DIRECTORY']}';
\$THEMES_DIRECTORY    = '{$this->e107->e107_dirs['THEMES_DIRECTORY']}';
\$PLUGINS_DIRECTORY   = '{$this->e107->e107_dirs['PLUGINS_DIRECTORY']}';
\$HANDLERS_DIRECTORY  = '{$this->e107->e107_dirs['HANDLERS_DIRECTORY']}';
\$LANGUAGES_DIRECTORY = '{$this->e107->e107_dirs['LANGUAGES_DIRECTORY']}';
\$HELP_DIRECTORY      = '{$this->e107->e107_dirs['HELP_DIRECTORY']}';
\$MEDIA_DIRECTORY	  = '{$this->e107->e107_dirs['MEDIA_DIRECTORY']}';
\$SYSTEM_DIRECTORY    = '{$this->e107->e107_dirs['SYSTEM_DIRECTORY']}';
```

However, I think the e107_core and the e107_web folders are missing to be made configurable.

**Comment by nlstart** [comment]
it is stated on https://e107.org/developer-manual/ in 'Files and Folders' that these folders cannot be modified; however, that is also stated for the e107_admin and the e107_handler folders, yet they are being offered to be made configurable.

**Comment by Jimmi08** [comment]
You can add to your config file this manually
```
$CORE_DIRECTORY = 'core/';
$WEB_DIRECTORY = 'web/';
```
And you need to change .htaccess too if you change Images and File folders.  

You shouldn't change handler and admin folders before installation, because names are hardcoded
![image](https://user-images.githubusercontent.com/5429548/55476115-fc609200-5615-11e9-80ec-c7ed296b642c.png)

By the way, where you found in documention that "that these folders cannot be modified" ?  I can see there only there changes inside those folders shouldn't be made.   

PS. Nice to see your nick here :)

**Comment by nlstart** [comment]
![image](https://user-images.githubusercontent.com/2721639/55501896-b32b3500-564b-11e9-969c-5a6443bbe458.png)

**Comment by CaMer0n** [comment]
@nlstart That means the **content** can be modified. It has nothing to do with the paths in e107_config.php. Defaults are set when paths are not found in this file.

**Comment by Jimmi08** [comment]
@nlstart I use non-prefix version on one site more than 2 years without issues.  CaMeRon answer is correct.

**Comment by nlstart** [comment]
> 
> 
> @nlstart That means the **content** can be modified. It has nothing to do with the paths in e107_config.php. Defaults are set when paths are not found in this file.

Makes sense now, wasn't so clear after reading the developer manual. Plus that not all configurable paths are placed in the initial e107_config.php by install.php.

**Comment by CaMer0n** [comment]
Ideally, I'd like to see them all commented out. (see last commit)
